packages/atomic/src/components/atomic-load-more-results/atomic-load-more-results.pcss

```diff
@@ -0,0 +1,16 @@
+@import '../../global/global.pcss';
+
+/**
+ * @prop --atomic-more-results-progress-bar-color-primary: Color of the start of the gradient for the load more results progress bar.
+ */
+.more-results-progress-bar-primary {
+  --tw-gradient-from: var(--atomic-more-results-progress-bar-color-primary, #0461d9);
```

Comment:
If possible, I would like if those variables defaulted to the primary color or a variation of it.

Reply:
+1